sympy/core/tests/test_numbers.py

```diff
@@ -121,13 +122,15 @@ def test_mod():
 
 
 def test_divmod():
+    s0 = Symbol("s0")
     assert divmod(S(12), S(8)) == Tuple(1, 4)
     assert divmod(-S(12), S(8)) == Tuple(-2, 4)
     assert divmod(S.Zero, S.One) == Tuple(0, 0)
     raises(ZeroDivisionError, lambda: divmod(S.Zero, S.Zero))
     raises(ZeroDivisionError, lambda: divmod(S.One, S.Zero))
     assert divmod(S(12), 8) == Tuple(1, 4)
     assert divmod(12, S(8)) == Tuple(1, 4)
+    assert divmod(1024,s0)[0] == floor(1024/s0)
```

Comment:
This test already passes on master. We need another test for the case that fails which is specifically `Integer(1024) // s0`. We should also test `1024 // s0` if that isn't tested anywhere.

The symbol should just be called `x` rather than `s0`.